This patch works around a name clash between the 'cstring-pointer' module and
the 'cstring-module' variable that occurs in Guile 2.0:

  ice-9/boot-9.scm:109:20: re-exporting local variable: cstring-pointer

--- guile-bytestructures-20170402.91d042e-checkout/bytestructures/guile.scm	2017-07-25 17:04:32.858289986 +0200
+++ guile-bytestructures-20170402.91d042e-checkout/bytestructures/guile.scm	2017-07-25 17:04:41.130244725 +0200
@@ -1,6 +1,6 @@
 (define-module (bytestructures guile))
 
-(import
+(use-modules
  (bytestructures guile base)
  (bytestructures guile vector)
  (bytestructures guile struct)
@@ -8,7 +8,7 @@
  (bytestructures guile pointer)
  (bytestructures guile numeric)
  (bytestructures guile string)
- (bytestructures guile cstring-pointer))
+ ((bytestructures guile cstring-pointer) #:prefix cstr:))
 (re-export
  make-bytestructure-descriptor
  bytestructure-descriptor?
@@ -75,5 +75,5 @@
 
  bs:string
 
- cstring-pointer
+ cstr:cstring-pointer
  )
